t/t7519/fsmonitor-watchman

```diff
@@ -64,8 +65,7 @@ sub launch_watchman {
 	my $query = <<"	END";
 		["query", "$git_work_tree", {
 			"since": $time,
-			"fields": ["name"],
-			"expression": ["not", ["allof", ["since", $time, "cclock"], ["not", "exists"]]]
+			"fields": ["name"]
```

Comment:
Can you update the above comment? This part doesn't appear to be true anymore:

`# To accomplish this, we're using the "since" generator to use the`

Reply:
> Could you provide a few more details on this scenario? Does "adds that are then removed" mean new files that are quickly created and deleted, or is it something to do with git add?

I think I just realized you were referring to this code, in that case, please ignore my earlier comment.

Reply:
Also, what is the downside of getting the names of files that were transient.  Is it just a perf issue?

Reply:
>what is the downside of getting the names of files that were transient. Is it just a perf issue?

Yes it would just mean that more file names would be returned that would need to be processed and in most cases these kind of files are ones that don't need to be because they were not in the index because they were new files (untracked) but now don't exist so won't be in the git status.